app/routes.rb

```diff
@@ -132,7 +132,7 @@ def notify_upsert_exam(exam_id)
 end
 
 require_relative './routes/courses'
-require_relative './routes/comments'
+require_relative './routes/messages'
 require_relative './routes/errors'
 require_relative './routes/exams'
 require_relative './routes/followers'
```

Comment:
We should use current_user_uid helper in all followers' routes instead of getting it from client request.